pyresttest/validators.py

```diff
@@ -1,5 +1,6 @@
 import logging
 import json
+import Cookie
```

Comment:
Fair, but we need to do the following for python 3 compatibility, since the library changed in Py3 :cry:  https://docs.python.org/2/library/cookie.html

if 'sys' is not in imports, add it:

```python
PYTHON_MAJOR_VERSION = sys.version_info[0]
if PYTHON_MAJOR_VERSION > 2:
    import http.cookies as Cookie
else:
   import Cookie
```